adapters/repos/db/shard_write_put.go

```diff
@@ -523,9 +524,16 @@ func (s *Shard) updateInvertedIndexLSM(object *storobj.Object,
 	}
 
 	before := time.Now()
-	if err := s.extendInvertedIndicesLSM(propsToAdd, nilpropsToAdd, status.docID); err != nil {
-		return fmt.Errorf("put inverted indices props: %w", err)
+	if prevObject != nil && os.Getenv("USE_INVERTED_SEARCHABLE") == "true" {
```

Comment:
could you add a brief comment describing this change? I'm wondering why props is used when `prevObject != nil && os.Getenv("USE_INVERTED_SEARCHABLE") == "true"` holds

Reply:
This is related to the behavior when dealing with patching and partial updates, I'll add a comment to describe my reasoning.